kalite/distributed/hbtemplates/exercise/exercise.handlebars

```diff
@@ -37,6 +38,7 @@
                                             {{_ "Loading..." }}
                                         </div>
                                     </div>
+                                    <div id="software-keyboard-container"></div>
```

Comment:
Attaching the software keyboard to the solutionarea (which had been the previous way it was done) had left it el-less after Khan Exercises deleted it.